src/applications/edu-benefits/0994/components/PaymentView.jsx

```diff
@@ -40,6 +50,9 @@ export const PaymentView = ({ formData = {}, originalData = {} }) => {
         <p>Account number: {maskBankInformation(bankAccountNumber, 4)}</p>
         <p>Bank routing number: {maskBankInformation(bankRoutingNumber, 4)}</p>
       </div>
+      {hasNewBankAccountInfo && (
+        <AlertBox isVisible status="success" content={directDepositAlert} />
```

Comment:
Similar to another PR earlier, you could pass `hasNewBankAccountInfo` to the `isVisible` prop and remove the conditional.